.gitignore

```diff
@@ -8,20 +8,26 @@ packages/hardhat/*.txt
 **/node_modules
 
 packages/hardhat/artifacts*
-packages/hardhat/deployments
+packages/hardhat/deployments/hardhat
+packages/hardhat/deployments/localhost
 packages/react-app/src/contracts/*
 !packages/react-app/src/contracts/external_contracts.js
```

Comment:
i don't think we need `react-app` anymore, its no longer `vite-app-ts`
neither hardhat is no longer, its `hardhat`